common/viewer.cpp

```diff
@@ -2102,7 +2028,7 @@ namespace rs2
 
                 glMultMatrixf((float*)_rx);
 
-                streams[f.get_profile().unique_id()].dev->tm2.draw_trajectory(trajectory_button.is_pressed());
+                streams[f.get_profile().unique_id()].dev->tm2.draw_trajectory( false );
```

Comment:
Why do we need tm2 struct to remain?

Reply:
It's named `tm2` but it's any pose data, really
Thought there might be usage elsewhere by other SKUs and this was more involved.